src/student.rb

```diff
@@ -0,0 +1,12 @@
+require './add'
+
+class Student < Person
+  def initialize(classroom)
```

Comment:
- [ ] I think here you missed some parameters so here when inheriting a class so also it inherit the paramters so please add the parameters for age, name and parent_permission with their defult values 💯🔥💪🏻😊

Reply:
@Reem-lab Hi, thank you for your review, Wasn't that what I did here?

![image](https://user-images.githubusercontent.com/97189760/199136411-8e260293-5d7a-4a31-86f1-28111c94dcb7.png)

Reply:
Hello @angeluray I think also you need to pass it with initialize method to be able to pass to the super() 💯😊

Reply:
- [ OPTIONAL ] I think you missunderstand me so it could be somthing like this

```ruby
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```